lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -76,6 +76,13 @@ def munge_boolean(value)
     raise ArgumentError.new "invalid value: #{value}"
   end
 
+  def munge_ensure(value)
+    return 'absent' if value.downcase =~ (/^(absent|disable)$/i)
```

Comment:
Unfortunately we won't have access to the type since the type for `powershell.rb` is `base_dsc.rb`. However if we move this INTO the type in the section where it creates a provider named powershell that inherits from powershell.rb, we can do it there.